src/System Application/App/Azure Blob Services API/src/ABSBlobClient.Codeunit.al

```diff
@@ -569,6 +569,18 @@ codeunit 9053 "ABS Blob Client"
         exit(ABSClientImpl.SetBlobTags(BlobName, Tags));
     end;
 
+    /// <summary>
+    /// The Find Blobs By Tags operation retrieves blobs based on user-defined tags for the specified blob, represented as one or more key-value pairs.
+    /// see: https://go.microsoft.com/fwlink/?linkid=2211407
```

Comment:
That link actually points to "Set Blob Tags" and not to "Find Blobs By Tags". Minor thing, but maybe worth fixing before we merge?

Reply:
I changed the URL to the Find Blobs By Tags documentation.